pkg/tests/api/plugins/api_plugins_test.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 const (
-	usernameAdmin    = "admin"
+	usernameAdmin    = "admin2"
```

Comment:
I wonder why this was needed?

Reply:
it is removed now, but because of my previous way I have injected sqlstore, admin user was already added